packages/esm-ward-app/src/hooks/usePatientPendingOrders.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { type Concept, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+import { type Encounter } from '../types';
+
+const encounterRepresentation =
+  'custom:(uuid,encounterDatetime,encounterType,location:(uuid,name),' +
+  'patient:(uuid,display),encounterProviders:(uuid,provider:(uuid,name)),' +
+  'obs:(uuid,obsDatetime,voided,groupMembers,formFieldNamespace,formFieldPath,order:(uuid,display),concept:(uuid,name:(uuid,name)),' +
+  'value:(uuid,display,name:(uuid,name),names:(uuid,conceptNameType,name))))';
+
+const conceptRepresentation =
+  'custom:(uuid,display,name,datatype,set,answers,hiNormal,hiAbsolute,hiCritical,lowNormal,lowAbsolute,lowCritical,units,' +
+  'setMembers:(uuid,display,answers,datatype,hiNormal,hiAbsolute,hiCritical,lowNormal,lowAbsolute,lowCritical,units))';
+
+export function useTestOrderCount(testOrders: Order[]) {
```

Comment:
What does this function do? It isn't clear to me why this takes a `testOrders` object but then still needs to make a bunch of requests, especially if all it's doing is counting the number of them that are test orders... which if the param is called `testOrders` should be all of them. This is all very confusing. Please write a docstring and consider some better names.